Automattic/jetpack
# Pre-flight checks: Add post build action step to trigger TC build

Adds a new step to a Build workflow to trigger TeamCity build for the PR. 

#### Other information:


#### Jetpack product discussion


pd5faL-eW-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Since the whole integration is in place already, you can see the added status check, which should include the link to the Phab diff and the tests status.
* One can re-run the build action to see the updated status check
